assets/src/application/ui/input/InlineEditInput/types.ts

```diff
@@ -1,30 +1,29 @@
 import { TextProps as AntdTextProps } from 'antd/es/typography/Text';
 import { BlockProps } from 'antd/es/typography/Base';
 
-interface EllipsisConfig {
-	rows?: number;
-	expandable?: boolean;
-	onExpand?: () => void;
-}
-
-export interface AdditionalProps {
+export interface AdditionalProps extends BlockProps, Omit<AntdTextProps, 'ellipsis'> {
+	ellipsis?: boolean | EllipsisConfig;
 	onChange?: (text: string) => void;
 }
 
-type BaseProps = Omit<BlockProps, 'editable'>;
+// type BaseProps = Omit<BlockProps, 'editable'>;
+
+export interface EditableProps extends HeadingProps, AdditionalProps {
+	inputType: InputType;
+	tabIndex?: number;
+}
 
-export interface TextProps extends AdditionalProps, Omit<AntdTextProps, 'ellipsis'> {
-	ellipsis: boolean | EllipsisConfig;
+interface EllipsisConfig {
+	rows?: number;
+	expandable?: boolean;
+	onExpand?: () => void;
 }
```

Comment:
this is all wonky now because the AntD typing is screaming about the ellipsis parameter...
I think Manzoor had this working but when I made changes to the inputs this all went crazy and blew up.
I tried to get it working again, but without luck....

( sorry Manzoor 😟 )

Reply:
Yeah, that was tricky when I dealt with it. You can remove that `ellipsis` prop above to make it wok.